### PR TITLE
If last price is missing use quote

### PR DESCRIPTION
**Pull request by jsom**

### CODE REVIEW

File: TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsViewController.swift [hunk 1]
```diff
@@ -76,32 +76,66 @@ class TradeItPortfolioAccountDetailsViewController: TradeItViewController, Trade
...
-            return when(fulfilled: accountOverviewPromise, positionsPromise)
-        }.catch { error in
-            print(error)
-        }.always {
-            onRefreshComplete()
+    private func quotesPromise(portfolioPositions: [TradeItPortfolioPosition]) -> Promise<[TradeItPortfolioPosition]> {
+        let symbols = portfolioPositions
+            .filter { $0.position?.lastPrice == nil }
+            .map { $0.position?.symbol }
+            .flatMap { $0 }
```
Comment (guillaumedebavelaere) [inline TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsViewController.swift]: Is it possible when position is nil to end with a nil object in the `symbols` array ?

**Comment by guillaumedebavelaere** [comment]
I wrote just a question otherwise it looks good

File: TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsViewController.swift (hunk 1 above)
Comment (jsom) [reply TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsViewController.swift]: The `flatMap` should remove it but I'll double check

Comment (jsom) [reply TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsViewController.swift]: I can probably combine the `map` and `flatMap` actually

**Pull request merged**